### ISSUE TITLE
Only 2 Pages when choosing creatures for expeditions

### ISSUE THREAD

**Issue by meldrian** [Varkalandar/imp_city#14]
Hey, another thing I noticed, in a kind of progressed game state with over 60 creatures in the dungeon (besides the imps), there are still max 2 pages available while assembling a party for an expedition. So, some of my low level creatures (I guess its sorted by level?) are not available in that screen at all.
However, when Party "A" is assembled and on its way to retrieve treasure, if I try to create another Party "B", some of the former not available creatures become visible in said screen but, again, only on page 1 and 2 since there's no page 3,4,5, etc.

tl;dr: no matter how many creatures a dungeon has, only the first/strongest 20 creatures of a dungeon are available for assembling an expedition party due the lack of pages greater than 2.

**Comment by Varkalandar** [comment]
I'm sorry for having so little time to actually work on the project currently. Next week I'll try to fix the problems that you have reported. 

Thanks again for this report. I seldom get to test such big and developed games.
